### DISCUSSION TITLE
How can I redirect the page on event (like: on function call , after  mounted  pages)

### DISCUSSION THREAD

**Discussion by khushmohammad** [tezjs/tezjs#109] [Q&A] [answered]
How can I redirect the page on event (like: on function call , after  mounted  pages)

**Comment by ajayojha** [comment, marked as answer]
In the component, you will get an instance of tez router ($tezRouter). Use `$tezRouter` method of `push`. Here is an example:

```js
this.$tezRouter.push('/custom-route')
```
